### PR TITLE
2 front decoration parts 

### PR DESCRIPTION
**Pull request by underpk**
19mm button and  Voron logo on gear fairing.

### CODE REVIEW

File: Mods/README.md [hunk 1]
```diff
@@ -337,4 +337,13 @@ Also covers the Z-belt hole for improved aesthetics.
 
 - Different mounts for Wago connectors.  
 
+## underpk
+
+[19mm_Button.stl](https://github.com/underpk/Voron-2/tree/master/Mods/underpk)
+- 19mm button mount for front skirt
+
+[Voron_logo_cap.stl](https://github.com/underpk/Voron-2/tree/master/Mods/underpk)
+- Add Voron logo on gear fairing
```
Comment (TorinoFermic) [inline Mods/README.md]: The link to other repo is not required when referencing these files. You instead use the `[Name of item where the stl is located](../mods/underpk)`
Then leave this link to your repo after these lines.

File: Mods/underpk/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## underpk
+
+[19mm_Button.stl](https://github.com/underpk/Voron-2/tree/master/Mods/underpk)
+- 19mm button mount for front skirt
+
+[Voron_logo_cap.stl](https://github.com/underpk/Voron-2/tree/master/Mods/underpk)
+- Add Voron logo on gear fairing
```
Comment (TorinoFermic) [inline Mods/underpk/README.md]: Ditto here.

**Comment by john--** [comment]
It appears there are some conflicts

**Comment by FHeilmann** [comment]
Hey, I had a look at your two parts and I quite like the concept.

Here are a few minor tweaks I would suggest:

- The tabs on the buttons were not long enough to 'reach around the backside' of the skirt for me by about .2 to .3mm.

- The tabs in general just consist of 1-2 unsupported lines, which makes them fragile and prone to just snap off on first insertion. Since you went with a pretty solid press fit, I would suggest to either remove them altogether or beefing them up slightly (while also supporting them with a proper chamfer)

Cheers,
-Florian

**Review by FHeilmann — CHANGES_REQUESTED** [review]
Hey, I had a look at your two parts and I quite like the concept.

Here are a few minor tweaks I would suggest:

The tabs on the buttons were not long enough to 'reach around the backside' of the skirt for me by about .2 to .3mm.

The tabs in general just consist of 1-2 unsupported lines, which makes them fragile and prone to just snap off on first insertion. Since you went with a pretty solid press fit, I would suggest to either remove them altogether or beefing them up slightly (while also supporting them with a proper chamfer)

Cheers,
-Florian

edit: Also if you still have them, could you add the step files/f3d files for this mod?

edit2: regarding the conflicts. It's probably better to close this PR and start with a fresh copy of the current master branch.

**Comment by FHeilmann** [comment]
We have moved all the mods to a new repository: https://github.com/VoronDesign/VoronUsers
Please resubmit your PR over there!

Thanks!

**Comment by FHeilmann** [comment]
I'm going to close this for now. Feel free to resubmit this PR to the VoronUsers repository!

**Pull request closed without merging**